source/renderer/webpack.config.js

```diff
@@ -83,15 +83,20 @@ module.exports = {
   plugins: [
     // Set the ExtractTextPlugin output filename
     new ExtractTextPlugin('styles.css', { allChunks: true }),
-    new webpack.DefinePlugin({
+    new webpack.DefinePlugin(Object.assign({
       'process.env.API': JSON.stringify(process.env.API || 'ada'),
       'process.env.API_VERSION': JSON.stringify(process.env.API_VERSION || 'dev'),
       'process.env.NETWORK': JSON.stringify(process.env.NETWORK || 'development'),
       'process.env.MOBX_DEV_TOOLS': process.env.MOBX_DEV_TOOLS || 0,
       'process.env.BUILD_NUMBER': JSON.stringify(process.env.BUILD_NUMBER || 'dev'),
       'process.env.REPORT_URL': JSON.stringify(reportUrl),
       'process.env.WALLET_PORT': JSON.stringify(process.env.WALLET_PORT || ''),
-    }),
+    }, process.env.NODE_ENV === 'production' ? {
+      // Only bake in NODE_ENV and WALLET_PORT values for production builds.
```

Comment:
@daniloprates we are only "baking" in `NODE_ENV` here so we should update this comment to be only:
```
// Only bake in NODE_ENV value for production builds.
```

Reply:
@nikolaglumac ✓